internal/bminventory/inventory.go

```diff
@@ -245,6 +255,15 @@ func (b *bareMetalInventory) getUserSshKey(params installer.GenerateClusterISOPa
 		"groups": [ "sudo" ]}`, sshKey)
 }
 
+func (b *bareMetalInventory) getIPs() []byte {
```

Comment:
I think there should be unit test for this function.

Reply:
Noted, will add.

Reply:
Done